### ISSUE TITLE
Breadcrumb items with no link have insufficient color contrast

### ISSUE THREAD

**Issue by tulup-conner** [themesberg/flowbite-react#102]
**Describe the bug**
The final item in a `Breadcrumb` is given `text-gray-400`, which is not accessible on the standard white background. The same goes for the dark mode, which uses `text-gray-500`.

**To Reproduce**
1. Create a `Breadcrumb`
2. Observe the contrast ratio of the final item

**Expected behavior**
This item should pass WCAG 2.1 AA, which designates a contrast ratio of 4.5:1 for normal text.

**Screenshots**
n/a

**Additional context**
n/a

**Comment by zoltanszogyenyi** [comment]
@robert1508 can you please take a look at this?

**Comment by robert1508** [comment]
Sure, I will check and fix this issue.

**Comment by zoltanszogyenyi** [comment]
@tulup-conner @rluders @robert1508 

After talking with Robert, we should settle on the following classes:

`text-gray-500` for the light version and `dark:text-gray-400` for the dark version.
